FFXIVClientStructs/FFXIV/Client/Game/MJIManager.cs

```diff
@@ -165,45 +162,40 @@ public unsafe partial struct MJIManager {
     ///     logic. To that end, this field doesn't actually seem authoritative for determining what's going on - see
     ///     <see cref="Granaries"/> and <see cref="Workshops"/> for what seems to be used by system logic.
     /// </remarks>
-    [FieldOffset(0x19C)] public MJIBuildingPlacements BuildingPlacements;
-    
+    [FieldOffset(0x1B4)] public MJIBuildingPlacements BuildingPlacements;
+
     /// <summary>
-    ///     The current day in the Craftworks cycle, from 0 to 6.
+    ///     A struct representing the cabin's placement, similar to the functionality of
+    ///     <see cref="MJIBuildingPlacements"/>.
     /// </summary>
     /// <remarks>
-    ///     0 represents reset day (Tuesday).
+    ///     Like <c>MJIBuildingPlacements</c>, the purpose of this field is unknown but it at least appears to behave
+    ///     like any other building placement.
     /// </remarks>
-    [FieldOffset(0x27A)] public byte CurrentCycleDay;
-    
+    [FieldOffset(0x204)] public MJIBuildingPlacement CabinPlacement;
+
     /// <summary>
-    ///     An array containing the currently-configured rest days for the Isleworks. Contains values 0 - 13, and is
-    ///     always in order.
+    ///     A struct representing farm (garden/cropland) placements on the current Island Sanctuary.
     /// </summary>
-    /// <remarks>
-    ///     Like CurrentCycleDay, 0 represents Reset Day. 7, likewise, represents the next reset. This field may not be
-    ///     populated until the Craftworks have been opened at least once.
-    /// </remarks>
-    [FieldOffset(0x27B)] public fixed byte CraftworksRestDays[4];
+    [FieldOffset(0x214)] public MJIFarmPasturePlacements FarmPlacements;
```

Comment:
Would a [FixedSizeArray<MJIFarmPasturePlacement>]` work for here?

Reply:
Most likely, I just converted all of the helper structs over.